config/locales/cy/cy.yml

```diff
@@ -378,6 +378,7 @@ cy:
         net_pay: "Hwn yw eich tâl net neu dâl terfynol, ar ôl treth a didyniadau eraill. Gallwch ddod o hyd iddo ar eich slip cyflog. Dylech gynnwys unrhyw daliadau goramser, comisiwn a bonws"
         benefit_details: Manylion budd(ion)
         end_date: "Er enghraifft, 22 04 2014 (os nad ydych yn gwybod yr union ddyddiad, rhowch eich amcangyfrif gorau)"
+        end_date_before_start: Rhaid nodi dyddiad sydd ar ôl dyddiad cychwyn y gyflogaeth
```

Comment:
Is this a mistake ?  The correct place is line 860 which you have - I doubt this one is used